.github/workflows/main.yml

```diff
@@ -0,0 +1,12 @@
+name: CI - On Push
+on: push
+jobs:
+  gitHubActionForPytest:
+    name: GitHub Action for pytest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: GitHub Action for pytest
+        uses: cclauss/GitHub-Action-for-pytest@master
+        with:
+          args: python setup.py develop && pytest
```

Comment:
This is a configuration for CI?

Reply:
Yes, this is a yaml file that is used by GitHub Actions. 

The location of the file is required by GitHub. I'm not sure if the file name `main.yml` is required to be as such. 

It creates an Action called "CI - On Push" that runs `on: push`, so every time a commit is pushed. 

This is using your allotted Action minutes per month, so might want to keep an eye on that value in your settings, as I have no idea how long these take. My guess is a trivial amount of time. 


I derived this one from this video:
https://www.youtube.com/watch?v=KgZZtA7_iGA&t=9s